schemas/Dockerfile

```diff
@@ -0,0 +1,78 @@
+#-------------------------
+FROM alpine:3.12.0 as file-loader
+
+# To preserve layer caching across machines which may have different local file properties
+# such as permissions, timestamps, etc, all files are copied into a container and their
+# permissions and timestamps are reset to consistent values
+# Credit: https://gist.github.com/kekru/8ac61cd87536a4355220b56ae2f4b0a9
+COPY . /schemas/
+RUN chmod -R 555 /schemas \
+    && chown -R root:root /schemas \
+    && find /schemas -exec touch -a -m -t 201512180130.09 {} \;
+
+
+#-------------------------
+# System packages
+FROM python:3.11.1 AS system-builder
+
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
+RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
+RUN apt-get update
+RUN apt-get --no-install-recommends install -y apt-utils ca-certificates postgresql-client yarn parallel
+
+# Install nvm with node and npm
+ENV NODE_VERSION=16.19.0
```

Comment:
This is EOL as of August. Maybe not worth worrying about on this PR but maybe worth a separate ticket (across the whole repo if needed).

Reply:
https://github.com/mozilla/experimenter/issues/9635